.github/workflows/pr.yaml

```
@@ -84,7 +81,7 @@ jobs:
           key: ${{ runner.os }}-rust-${{ env.RUST_VERSION_HASH }}-cargo-${{ hashFiles('./Cargo.lock', './crates/txtx-cli/build.rs', 'addons/evm/src/contracts/**') }}
 
       - name: Build Project
-        run: cargo build --all --release
+        run: cargo build --workspace --exclude txtx-addon-network-stacks --release
 
       - name: Run Tests
-        run: cargo test --all
+        run: cargo --workspace --exclude txtx-addon-network-stacks 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)